### ISSUE TITLE
When i try to send anything, it lets me but but the file is corrupted and "transfer complete" keeps popping up continuously (both desktop and phone) Chrome 92

### ISSUE THREAD

**Issue by iraouf** [RobinLinus/snapdrop#331]
https://vimeo.com/578271586

**Comment by fm-sys** [comment]
I can reproduce this behavior as well... Something seems to be really buggy since the latest changes.

**Comment by Len-B** [comment]
I reported this as https://github.com/fm-sys/snapdrop-android/issues/118 but was advised to report to this forum. I have included both my previous reports separated by ======.

Crashes with 'Aw, Snap!' when trying to save.

This problem has just started after a recent Google Chrome update. When trying to transfer files from my PC to an Android device they appear to transfer but then the save fails.

Yesterday I overcame the problem by using MS Edge and all went well. Today my MS Edge also had an update and now that also fails.

I then tried Firefox Quantum and that still works.

=============================

I have sent 3 screenshots, all from my Android app, but they don't show much more than I've descibed above.

I should have made it clear that the browser updates were on my Windows 10 Pro PC and both are now at the latest version.

I tried a browser to browser (PC Chrome to Android Chrome) transfer with the same result.

The progress circle completes it's full 360 and then there is a short hang before the 'Aw, Snap!' error occurs.
Reloading at both ends often results in loss of connection name at the PC end and sometimes duplicate names at the Android end.

Hope this is helpful,

Len

![image](https://user-images.githubusercontent.com/35223209/126881352-1abede5d-6109-4960-90f4-87cab083e5dd.png)
![image](https://user-images.githubusercontent.com/35223209/126881359-e1602775-9a78-4255-b7db-525664bf8f01.png)
![image](https://user-images.githubusercontent.com/35223209/126881368-b4d18e81-d687-4572-9eb1-8ca68bdf5c3f.png)
